copy.go

```diff
@@ -136,15 +135,10 @@ func (cp BucketCopier) optimizeStorageClass(size int64) string {
 	return s3.StorageClassStandard
 }
 
-func (cp *BucketCopier) copyFile(file fileJob, singleFile bool) {
```

Comment:
This `singleFile` argument is unnecessary.  I tested all s3 options to verify (recursive, and non-recursive single-file copies)